views/administration/forms/form-invitation-student.hbs

```diff
@@ -1,10 +1,10 @@
 <input name="schoolId" type="hidden" data-force-value="true" value="{{currentSchool}}" />
 <div class="form-group">
-    <label for="invitation-link">Um in die Schul-Cloud einzuladen, verteile folgenden Link an: </label>
+    <label for="invitation-link">{{$t "administration.label.toInviteToTheSchoolCloud" }} </label>
     <p>
-        <b>Schüler unter {{CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS}} Jahre:</b> die Eltern <br/>
-        <b>Schüler ab {{CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS}} Jahre:</b> an die Schüler selbst <br/>
+        <b>{{$t "administration.global.text.studentUnder" }} {{CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS}} {{$t "administration.form_InvitationStudent.text.years" }}</b> {{$t "administration.form_InvitationStudent.text.theParents" }} <br/>
+        <b>{{$t "administration.form_InvitationStudent.text.studentsFrom" }} {{CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS}} {{$t "administration.form_InvitationStudent.text.years" }}</b> {{$t "administration.form_InvitationStudent.text.toTheStudentsThemselves" }} <br/>
```

Comment:
here again use the interpolation function and add it without stripped html {{{ }}}

Reply:
please also do this everywhere else